domain/model/mesh_results.py

```diff
@@ -73,12 +73,12 @@ def is_no_data(self) -> bool:
         return self.packet_loss_percent.value == MetricValue(100) or self.health == []
 
 
-@dataclass
 class MeshRow:
-    """ Represents connection "from" endpoint """
+    """Represents connection "from" endpoint"""
 
-    agent_id: AgentID
-    columns: List[MeshColumn]
+    def __init__(self, agent_id: AgentID, columns: List[MeshColumn]):
+        self.agent_id = agent_id
+        self.columns = sorted(columns, key=lambda x: x.agent_id)
```

Comment:
OK, so now columns and rows are stored in the same order

Reply:
Yes, that is the whole point. The presentation code (AFAICT) assumes that.